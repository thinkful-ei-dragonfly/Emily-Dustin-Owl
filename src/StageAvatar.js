import React from 'react';


export default function(props) {
  if (props.onStage === true) {
  return (
    <div className="onstage-part">
      <h3>{props.name}</h3>
      <img src={props.avatar} alt='user avatar'/>
      <div className='icons'>
      </div>
    </div>
  )
  }
  else {
    return null;
  }
}