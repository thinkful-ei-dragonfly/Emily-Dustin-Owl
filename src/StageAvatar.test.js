import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import StageAvatar from './StageAvatar';



describe('List component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<StageAvatar
      key={4} name={'name'} 
      avatar={'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1'}
      onStage={true}/>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expexted', () => {
    const tree = renderer
    .create(<StageAvatar key={4} 
        name={'name'} 
        avatar={'https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1'}
        onStage={true} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

});