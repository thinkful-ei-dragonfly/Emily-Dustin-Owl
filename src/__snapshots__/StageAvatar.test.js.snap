// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component renders the UI as expexted 1`] = `
<div
  className="onstage-part"
>
  <h3>
    name
  </h3>
  <img
    alt="user avatar"
    src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
  />
  <div
    className="icons"
  />
</div>
`;
